src/color.rs

```diff
@@ -9,6 +9,17 @@ pub enum Color {
     White,
 }
 
+/// Only for the Black/White/Color-Displays
+#[derive(Clone, Copy, PartialEq, Debug)]
+pub enum TriColor {
+    /// Black color
+    Black,
+    /// White color
+    White,
+    /// Red color
```

Comment:
Make it neutral to somehow combine yellow and red

Reply:
Hmm yeah you're right that `Colored` has certain negative connotations.
Going off the [wikipedia page for Colorfulness](https://en.wikipedia.org/wiki/Colorfulness) I suppose the most correct name would be `Chromatic` although `Colorful` is good too

Reply:
Sounds good